src/state/slices/backdrop_slice.ts

```diff
@@ -80,15 +91,23 @@ export const addBackdropDerivedStateSubscribers = (store: SubscribableStore<Back
   );
 };
 
-export const serializeBackdropSlice = (state: BackdropSlice): SerializedStoreData => {
+export const serializeBackdropSlice = (state: Partial<BackdropSliceSerializableState>): SerializedStoreData => {
```

Comment:
This changed to `Partial` because sometimes we turn a bit of state into a URL, without including definitions for every serializable value in state.

For example, on the landing page, I create URLs using partial states, containing just the collection URL and the starting frame number.